test/athens/common_events/block_test.clj

```diff
@@ -776,7 +777,288 @@
+                              :block/children [{:db/id          -2
+                                                :block/uid      target-parent-uid
+                                                :block/string   target-parent-str
+                                                :block/order    0
+                                                :block/children [{:db/id          -3
+                                                                  :block/uid      target-uid
+                                                                  :block/string   target-text
+                                                                  :block/order    0
+                                                                  :block/children []}
+                                                                 {:db/id          -4
+                                                                  :block/uid      source-uid
+                                                                  :block/string   source-text
+                                                                  :block/order    1
+                                                                  :block/children []}]}]}]]
+
+
+      (d/transact @fixture/connection setup-txs)
+      (let [source-block            (common-db/get-block @@fixture/connection [:block/uid source-uid])
+            target-block            (common-db/get-block @@fixture/connection [:block/uid target-uid])
+            target-parent-block     (common-db/get-block @@fixture/connection [:block/uid target-parent-uid])
+            drop-same-parent-event  (common-events/build-drop-same-event -1
+                                                                         :above
+                                                                         source-uid
+                                                                         target-uid)
+            drop-same-parent-txs    (resolver/resolve-event-to-tx @@fixture/connection drop-same-parent-event)]
+        (t/is (= 2 (-> target-parent-block :block/children count)))
+        (t/is (= 0 (:block/order target-block)))
+        (t/is (= 1 (:block/order source-block)))
+
+        (d/transact @fixture/connection drop-same-parent-txs)
+        (let [source-block         (common-db/get-block @@fixture/connection [:block/uid source-uid])
+              target-block         (common-db/get-block @@fixture/connection [:block/uid target-uid])
+              target-parent-block  (common-db/get-block @@fixture/connection [:block/uid target-parent-uid])]
+          (t/is (= 2 (-> target-parent-block :block/children count)))
+          (t/is (= 1 (:block/order target-block)))
+          (t/is (= 0 (:block/order source-block))))))))
+
+
+(t/deftest drop-multi-same-all-test
+  "Basic Case:
+       -a
+         -b
+         -c
+         -d
+     End:
+       -a
+         -c
+         -d
+         -a"
+  (t/testing "Drop multiple blocks inside the same source parent "
```

Comment:
If you don't have multiple `t/testing` blocks inside `t/deftest` it isn't needed at all.

Reply:
Yeah but currently we have only basic tests, maybe in future we will add more tests so is it not better to let `t/testing` be there in the codebase or this gives the wrong impression of multiple tests already been written?